README.md

```diff
@@ -202,4 +202,36 @@ from graphql_ws.django_channels import GraphQLSubscriptionConsumer
 channel_routing = [
     route_class(GraphQLSubscriptionConsumer, path=r"^/subscriptions"),
 ]
-```
\ No newline at end of file
+```
+
+### Tornado
+```python
+from asyncio import Queue
+from tornado import web, ioloop, websocket
+
+from graphql_ws.tornado import TornadoSubscriptionServer
+
+
+subscription_server = TornadoSubscriptionServer(schema)
+
+
+class SubscriptionHandler(websocket.WebSocketHandler):
+    def initialize(self, sub_server):
+        self.subscription_server = subscription_server
```

Comment:
I am trying the example locally, but this part was a bit confusing. We have the `subscription_server ` defined before this class declaration.

Here we get `self.subscription_server  = subscription_server `, but we also get a `sub_server` parameter?

The `open()` method uses not `self.subscription_server `, but the previously created `subscription_server `.